src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color $duration;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
Here too, pointer should be in class, not pseudo-class

Reply:
done